Umbraco-Cloud/Frequently-Asked-Questions/index.md

```diff
@@ -160,13 +160,11 @@ Learn more about how to use your own certificates in the [Custom certificates](.
 
 ### Does Umbraco Cloud support http/2?
 
-The lowest version of IIS to support http/2 is version 10, which runs only on Windows Server 2016. Currently our infrastructure is limited to Windows Server 2012 R2 instances and as such we do not support http/2 directly.
-
-As a workaround, you could consider setting up a product like CloudFlare, which offers free support for http/2 (they call it "Opportunistic Encryption") out of the box.
+On the server, IIS 10.0 running on Windows 10 or Windows Server 2016 supports HTTP/2. Umbraco runs on Azure Web Apps which provides support for the HTTP/2 protocol for all applications hosted on the Azure App Service.
```

Comment:
We support http/2 using Cloudflare, this is an implementation detail though. I'd just say we support HTTP/2 by default.